### PR TITLE
Update QUEST.tsv

### PR DESCRIPTION
**Pull request by rangarus**
Fixed translation from 1 to 161.

### CODE REVIEW

**Comment by JulioCF** [comment]
Very nice, thanks =D

File: QUEST.tsv [hunk 1]
```diff
@@ -1,164 +1,164 @@
...
-QUEST_20150317_000022	Essa tarefa divina pode ser um fardo pesado, mas ?algo que voc?deve suportar. {nl}Talvez no decorrer enlameado de suas provaes e dificuldades algo venha a surgir.
-QUEST_20150317_000023	Entretanto, s?n? podemos salvar esse mundo.
-QUEST_20150317_000024	Eu passei a pr?ima revelao ao primeiro Paladino. {nl}Ele espera por voc?no mais alto jardim florido.
-QUEST_20150317_000025	A aura escura realmente ?desagrad?el... {nl}Mas se essa mensagem realmente for da Deusa Laima, estou completamente tomado por admirao.
-QUEST_20150317_000026	Bem, voc?deveria se apressar e encontrar Sir Uska. {nl}Ele vai dizer para onde voc?deve seguir. {nl}Que a deusa aben?e seu destino...
+QUEST_20150317_000015	Quando você tiver recebido essa mensagem, muito tempo terá se passado. {nl}Sou Laima, a deusa do destino e sabedoria.
+QUEST_20150317_000016	600 anos atráz, enquanto criava essa revelação, previ um evento com muita morte e sofrimento. {nl}Se você está vendo isso, temo que minha profecia tenha se concretizado.
```
Comment (yalmeida69) [inline QUEST.tsv]: atrás com S

File: QUEST.tsv [hunk 2]
```diff
@@ -1,164 +1,164 @@
...
-QUEST_20150317_000058	At?quatro anos atr?, as deusas nos aben?avam com sinais de suas presen?s frequentemente. {nl}Mas agora, mesmo nossas preces combinadas n? t? o menor impacto. {nl}Ser?que elas realmente nos abandonaram?
-QUEST_20150317_000059	Vacenin vai pessoalmente a cada regi? para lidar com as coisas. {nl}Ele fica desconfort?el quando permanece na cidade.
-QUEST_20150317_000060	Voc?planeja ficar ou se alistar? {nl}Se tiver interesse, terei prazer em recomendar voc?aos comandantes.
-QUEST_20150317_000061	N? baixe sua guarda. Monstros podem aparecer a qualquer instante, em qualquer lugar.
-QUEST_20150317_000062	Onde ser?que a Deusa Zemyna foi parar...
-QUEST_20150317_000063	Como voc?chamaria uma coisa dessas?
+QUEST_20150317_000057	No passado, peregrinos viajavam por essa floresta. {nl}Mas agora, devido à presença de monstros, apenas soldados se reúnem aqui.
+QUEST_20150317_000058	Até quatro anos atráz, as deusas nos abençoavam com sinais de suas presenças frequentemente. {nl}Mas agora, mesmo nossas preces combinadas não têm o menor impacto. {nl}Será que elas realmente nos abandonaram?
```
Comment (yalmeida69) [inline QUEST.tsv]: atrás com S

File: QUEST.tsv [hunk 3]
```diff
@@ -1,164 +1,164 @@
...
-QUEST_20150317_000104	Achei que as vis?s dos Reveladores eram loucura, a princ?io. {nl}As deusas se foram faz muito tempo, afinal de contas.
-QUEST_20150317_000105	N? ?verdade que as deusas n? respondem ? nossas preces desde Medzio Diena, quatro anos atr??
-QUEST_20150317_000106	Voc?pode encontrar Mestres de Classe por todo o reino. {nl}Que tal visit?los e aprender o que for poss?el?
-QUEST_20150317_000107	Voc?n? sabe sobre Medzio Diena? Voc?deve ter vindo de muito longe mesmo. {nl}Quatro anos atr?, as coisas sa?am de controle na capital.
+QUEST_20150317_000101	Há monstros daqui em diante. Tenha cuidado.
+QUEST_20150317_000102	Essa floresta não é exatamente pacífica, mas é melhor que outros lugares. {nl}Há florestas com nada além de almas perambulando por elas.
+QUEST_20150317_000103	É disso que as pessoas que viram deusas em seus sonhos estão sendo chamadas.
+QUEST_20150317_000104	Achei que as visões dos Reveladores eram loucura, a princípio. {nl}As deusas se foram faz muito tempo, afinal de contas.
+QUEST_20150317_000105	Não é verdade que as deusas não respondem às nossas preces desde Medzio Diena, quatro anos atráz?
```
Comment (yalmeida69) [inline QUEST.tsv]: atrás com S

File: QUEST.tsv [hunk 4]
```diff
@@ -1,164 +1,164 @@
...
+QUEST_20150317_000106	Você pode encontrar Mestres de Classe por todo o reino. {nl}Que tal visitá-los e aprender o que for possível?
+QUEST_20150317_000107	Você não sabe sobre Medzio Diena? Você deve ter vindo de muito longe mesmo. {nl}Quatro anos atráz, as coisas saíram de controle na capital.
```
Comment (yalmeida69) [inline QUEST.tsv]: atrás com S

**Pull request closed without merging**